### ISSUE TITLE
vmalert runtime issue

### ISSUE THREAD

**Issue by vaclandic** [VictoriaMetrics/VictoriaMetrics#492]
**Describe the bug**
I am get 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x7c1621]
when starting docker container

**To Reproduce**
```sh
docker run -it --net=host -v /root/alert.rules:/alert.rules victoriametrics/vmalert:v1.35.4-cluster -rule=/alert.rules -datasource.url=https://xxxx -notifier.url=
http://x.x.x.x:9093
```
alert.rules was taken from examples 

**Version**
```
build version: vmalert-20200512-203449-tags-v1.35.4-cluster-0-g3b0f66a2
```

Log with trace in attach
[runtime.log](https://github.com/VictoriaMetrics/VictoriaMetrics/files/4635243/runtime.log)

**Comment by hagen1778** [comment]
duplicate https://github.com/VictoriaMetrics/VictoriaMetrics/issues/473

**Comment by valyala** [comment]
The bugfix has been included in [v1.35.5 release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
